src/components/tests/tour.test.tsx

```diff
@@ -0,0 +1,223 @@
+import { createMount } from '@material-ui/core/test-utils';
+import React from 'react';
+import DashboardTour from '../tour';
+import { SpotlightValue } from '../spotlight-value';
+import { ReactWrapper } from 'enzyme';
+import { OmniPanelViewIndex } from '../dashboard';
+
+/*react-leaflet does not work well on jsdom,
+suppress warnings of not being able to find elements from leaflet*/
+jest.spyOn(console, 'warn').mockImplementation(() => {});
+
+const mount = createMount();
+
+describe('Dashboard Tour', () => {
+  let tourState: boolean = true;
+  let setTourState: jest.Mock = jest.fn();
+  let setTourShowOmniPanel: jest.Mock = jest.fn();
+  let setTourSettingsAndOmniPanel: jest.Mock = jest.fn();
+  let doorSpotlight: SpotlightValue<string> = { value: 'main_door' };
+  let setDoorSpotlight: jest.Mock = jest.fn();
```

Comment:
Can those be `const`? Since you're not assign values to those after its definition?